substrait_consumer/tests/functional/relations/set_relation_snapshots/IsthmusProducer/intersect_plan.json

```diff
@@ -131,7 +131,7 @@
               }]
             }
           }],
-          "op": "SET_OP_INTERSECTION_PRIMARY"
+          "op": "SET_OP_INTERSECTION_MULTISET"
```

Comment:
Is this right behavior?   (It may be the existing behavior but it may also be wrong.)

Reply:
Good point! I haven't done extensive checks as to whether or not the updates makes sense.

In this specific case, I think that the two are equivalent: "primary" keeps records "that are present in *any* secondary input" and "multiset" keeps records that "that match at least one record from *all* secondary inputs"; since there is only a single secondary input in this query, "any" and "all" is the same.